### ISSUE TITLE
CAPTCHA: When text size is increased in Firefox, some content overlap/cut off

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#170]
**Page:** All (CAPTCHA under Report a problem or mistake on this page)
**URL:** https://recalls-rappels.canada.ca/en
**User Agent:** Firefox (v99)
**Language:** English and French

**Issue Description:** In Firefox, when text size is increased up to 200% using zoom text only, the error message and some content overlap/cut off and became unreadable.

![image](https://user-images.githubusercontent.com/12256067/162804424-e791bb12-4941-47d3-9856-62a6da80432c.png)

**Steps to reproduce:**

1. Open the above URL in Firefox
2. In Firefox, go to "View" in browser menu bar, select View - Zoom - Zoom Text Only  
4. Press Ctrl and + keys to increase zoom of text up to 200%
5. Access the "Report a problem or mistake on this page" component
6. Select the checkbox under CAPTCHA
7. Wait for the error message to appear

**Expected results:** All text should be able to be resized up to 200% without loss of content or functionality for text only zoom in Firefox.

**Comment by YuxuanXie01** [comment]
Retested, issue persist. However, we don't test Text only Zoom anymore, ticket should be closed.
![image](https://github.com/user-attachments/assets/6e41483c-25be-4611-9b91-4bcb90eaa8e5)

**Comment by 40999221** [comment]
Issue is still exist. Don't test on "text only resize" but zoom the page to 200% brings overlapping text. as shown below:

![image](https://github.com/user-attachments/assets/17df8d74-9a63-4968-ac8b-002a5e75f028)

**Comment by zimw7** [comment]
Issue still exists. Use 200% page zoom to see overlapping text. We no longer test text only zoom.
![image](https://github.com/user-attachments/assets/48961ede-42bb-4d8e-9a1e-2a94cb4191d2)

**Comment by mmiron-HC** [comment]
@40999221 Can you please create this ticket in ART. Please indicate that this is a product limitation with Google API. Mark ART ticket as Escalated
